src/paper/tasks.py

```diff
@@ -247,6 +248,44 @@ def add_orcid_authors(paper_id):
     return True
 
 
+@app.task(queue=QUEUE_PAPER_MISC)
+def add_orcid_authors_batch(paper_ids):
```

Comment:
that's a fair point, I kind of just copied the existing orcid authors add code and batched it.

I'm going to deploy this current version and then make the changes you suggested tomorrow, since this is technically existing logic that's been live for a while.